keps/sig-apps/NNNN-backoff-limits-per-index-for-indexed-jobs/README.md

```diff
@@ -0,0 +1,679 @@
+  - [Monitoring Requirements](#monitoring-requirements)
+  - [Dependencies](#dependencies)
+  - [Scalability](#scalability)
+  - [Troubleshooting](#troubleshooting)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+- [Infrastructure Needed (Optional)](#infrastructure-needed-optional)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input (including test refactors)
+  - [ ] e2e Tests for all Beta API Operations (endpoints)
+  - [ ] (R) Ensure GA e2e tests meet requirements for [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+  - [ ] (R) Minimum Two Week Window for GA e2e tests to prove flake free
+- [ ] (R) Graduation criteria is in place
+  - [ ] (R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806) must be hit by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+This KEP extends the indexed job API to support indexed jobs where each index is independent,
+and a failed index does not cause the other indices to automatically clean up.
+
+
+## Motivation
+
+Currently, the indices of an indexed job share a single backoff limit. 
+When the job reaches this shared backoff limit, the job controller marks the entire
+job as failed, and the resources are cleaned up, including indices that have yet
+to run to completion. 
+
+This implementation does not cover the situation where the workload is truly embarrassingly
+parallel and each index is completely independent of other indices. For instance, if
+indexed jobs were used as the basis for a suite of long-running integration tests,
+then each test run would only be able to find a single test failure.
+
+More generally, this use case is for any situation when all the indices in an indexed job 
```

Comment:
I find this comment to be a bit confusing.  You mention above that the implementation does not cover the situation where the workload is truly embarrassingly parallel.  

But then you also mention that the use case is for when all indices are independent.  I always thought that was what embarrassingly parallel meant all the cases were independent?

Reply:
Thank you for the feedback, I absolutely agree that it's confusing. I meant that the current implementation does not cover the use case when all indices are independent/embarassingly parallel, and that this KEP would cover it.

I've pushed a revised version that hopefully clears it up